--- awscli-1.16.167-orig/setup.py	2019-05-29 00:37:16.074441413 +0000
+++ awscli-1.16.167/setup.py	2019-05-29 01:00:07.938855836 +0000
@@ -24,11 +24,11 @@ def find_version(*file_paths):
 
 
 requires = ['botocore==1.12.157',
-            'colorama>=0.2.5,<=0.3.9',
+            'colorama>=0.2.5',
             'docutils>=0.10',
             'rsa>=3.1.2,<=3.5.0',
             's3transfer>=0.2.0,<0.3.0',
-            'PyYAML>=3.10,<=3.13']
+            'PyYAML>=3.10']
 
 
 if sys.version_info[:2] == (2, 6):
