diff -ur botocore-1.12.135.orig/botocore.egg-info/requires.txt botocore-1.12.135/botocore.egg-info/requires.txt
--- botocore-1.12.135.orig/botocore.egg-info/requires.txt	2019-04-24 15:41:11.000000000 -0400
+++ botocore-1.12.135/botocore.egg-info/requires.txt	2019-05-26 03:29:59.159464619 -0400
@@ -1,6 +1,6 @@
 jmespath<1.0.0,>=0.7.1
 docutils>=0.10
-python-dateutil<3.0.0,>=2.1
+python-dateutil<3.0.0,>=1.4
 urllib3<1.25,>=1.20
 
 [:python_version=="2.6"]
Only in botocore-1.12.135/botocore.egg-info: requires.txt~
diff -ur botocore-1.12.135.orig/requirements.txt botocore-1.12.135/requirements.txt
--- botocore-1.12.135.orig/requirements.txt	2019-04-24 15:38:24.000000000 -0400
+++ botocore-1.12.135/requirements.txt	2019-05-26 03:30:28.653869593 -0400
@@ -1,6 +1,6 @@
 tox>=2.5.0,<3.0.0
-python-dateutil>=2.1,<2.7.0; python_version=="2.6"
-python-dateutil>=2.1,<3.0.0; python_version>="2.7"
+python-dateutil>=1.4,<2.7.0; python_version=="2.6"
+python-dateutil>=1.4,<3.0.0; python_version>="2.7"
 nose==1.3.7
 mock==1.3.0
 wheel==0.24.0
diff -ur botocore-1.12.135.orig/setup.cfg botocore-1.12.135/setup.cfg
--- botocore-1.12.135.orig/setup.cfg	2019-04-24 15:41:12.000000000 -0400
+++ botocore-1.12.135/setup.cfg	2019-05-26 03:34:10.893921082 -0400
@@ -3,8 +3,8 @@
 
 [metadata]
 requires-dist = 
-	python-dateutil>=2.1,<2.7.0; python_version=="2.6"
-	python-dateutil>=2.1,<3.0.0; python_version>="2.7"
+	python-dateutil>=1.4,<2.7.0; python_version=="2.6"
+	python-dateutil>=1.4,<3.0.0; python_version>="2.7"
 	jmespath>=0.7.1,<1.0.0
 	docutils>=0.10
 	ordereddict==1.1; python_version=="2.6"
diff -ur botocore-1.12.135.orig/setup.py botocore-1.12.135/setup.py
--- botocore-1.12.135.orig/setup.py	2019-04-24 15:38:24.000000000 -0400
+++ botocore-1.12.135/setup.py	2019-05-26 03:34:40.071321706 -0400
@@ -38,9 +38,9 @@
     # JSON objects.  The 2.7 json module has this.  For 2.6
     # we need simplejson.
     requires.append('simplejson==3.3.0')
-    requires.append('python-dateutil>=2.1,<2.7.0')
+    requires.append('python-dateutil>=1.4,<2.7.0')
 else:
-    requires.append('python-dateutil>=2.1,<3.0.0')
+    requires.append('python-dateutil>=1.4,<3.0.0')
 
 if sys.version_info[:2] == (2, 6):
     requires.append('urllib3>=1.20,<1.24')
